.github/workflows/go.yml

```diff
@@ -93,6 +101,11 @@ jobs:
         run: go test -cover ./...
         working-directory: ./kyaml
 
+      # TODO: uncomment once Windows tests are passing.

```

Comment:
An issue should be created to fix these tests.

Reply:
filed https://github.com/kubernetes-sigs/kustomize/issues/4001